evm/vm/forks/byzantium/vm_state.py

```diff
@@ -0,0 +1,22 @@
+from evm.rlp.receipts import (
+    Receipt,
+)
+from evm.vm.forks.frontier.vm_state import _make_frontier_receipt
+from evm.vm.forks.spurious_dragon.vm_state import SpuriousDragonVMState
+
+from .computation import ByzantiumComputation
+
+
+class ByzantiumVMState(SpuriousDragonVMState):
+    computation_class = ByzantiumComputation
+
+    @staticmethod
+    def make_receipt(vm_state, transaction, computation):
+        old_receipt = _make_frontier_receipt(vm_state, transaction, computation)
+
+        receipt = Receipt(
+            state_root=b'' if computation.is_error else b'\x01',
```

Comment:
What does b'\x01' mean here? Should it be a constant?

Reply:
It was copied from `evm.vm.forks.byzantium.blocks`:
https://github.com/ethereum/py-evm/blob/master/evm/vm/forks/byzantium/blocks.py#L30

This protocol change is from [EIP 98/658](https://github.com/Arachnid/EIPs/blob/d1ae915d079293480bd6abb0187976c230d57903/EIPS/eip-658.md) Embedding transaction status code in receipts, 1 for success, 0 for failure.

Reply:
I think I support moving these two values into named constants to improve readability.